### ISSUE TITLE
switchTile subtitle not work on IOS

### ISSUE THREAD

**Issue by azazadev** [yako-dev/flutter-settings-ui#45]
Hi,
the switchTile subtitle no work on IOS, I'm testing with Example provided in package, did i forget something ?

```
SettingsTile.switchTile(
                title: 'Lock app in background',
                subtitle: 'subtitle test',
                leading: Icon(Icons.phonelink_lock),
                switchValue: lockInBackground,
                onToggle: (bool value) {
                  setState(() {
                    lockInBackground = value;
                    notificationsEnabled = value;
                  });
                },
              ),
```
attached screenshot for ISO / Android

![Screenshot 2020-12-25 at 18 53 51](https://user-images.githubusercontent.com/68111176/103140132-3378b000-46e3-11eb-9f03-0d640f582f14.png)

**Comment by Akora-IngDKB** [comment]
It does work.
It is shown on the right (as the `trailing` of some sort) instead of below the title as found in the Android image.

**Comment by Akora-IngDKB** [comment]
Thank you for the feedback.
We will look into it and fix the issue.

**Comment by RusKryzhanovskiy** [comment]
Hi, @azazadev, sorry for the delay. It will be available in version `0.6.0`
